### PR TITLE
rfs: T4689: Detaches global RFS from device for better control

### PR DESCRIPTION
**Pull request by jack9603301**
## Change Summary

Detaches global RFS from device for better control

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://phabricator.vyos.net/T4689

## Component(s) name

rfs

## Proposed changes

If the command line is not separated, there may be interference due to the configuration sequence, resulting in some devices using RFS, but the global RFS has been turned off

## How to test

图片

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/conf_mode/system-option.py [hunk 1]
```diff
@@ -132,6 +132,11 @@ def apply(options):
     else:
       cmd('systemctl disable root-partition-auto-resize.service')
 
+    if 'enable-rfs-option' in options:
+        cmd('echo 32768 > /proc/sys/net/core/rps_sock_flow_entries')
```
Comment (sever-sever) [inline src/conf_mode/system-option.py:136]: Can we use `sysctl -w`?
```
sysctl -w net.core.rps_sock_flow_entries=32768
```

**Comment by sever-sever** [comment]
Note. RFS cannot function if LRO is enabled

File: src/conf_mode/system-option.py (hunk 1 above)
Comment (jack9603301) [reply src/conf_mode/system-option.py:136]: Can we use sysctl -w?

Comment (jack9603301) [reply src/conf_mode/system-option.py:136]: I think it can, but sysctl -w will write to the sysctl config file putting

**Comment by c-po** [comment]
A more pragamtic approach is implemented in 5fe0e9c163ee2f8229e298fc20dbfe6746c2cdcc - thanks

**Pull request closed without merging**